common/src/light_client.rs

```diff
@@ -42,10 +37,20 @@ impl LightClient {
     /// Verifies the given data with its proof.
     pub fn verify_commitment(
         &self,
-        _message: Vec<u8>,
-        _block_height: u64,
-        _proof: MerkleProof,
+        message: Vec<u8>,
+        block_height: u64,
+        proof: MerkleProof,
     ) -> bool {
```

Comment:
Why is bool better than Result for return value?

Reply:
Because there is nothing to handle if verification fails. (And furthermore, this will be used in contract, so simple is better)